extensions/web/12_location.js

```diff
@@ -362,8 +362,9 @@
     locationDescriptor: {
       get() {
         if (location == null) {
-          throw new ReferenceError(
-            `Access to "location", run again with --location <href>.`,
+          console.warn(
+            "Warning: accessing undefined 'location' global, " +
+              "run again with --location",
```

Comment:
This seems unfortunate... `typeof location !== "undefined"` would result in a warning logged to the console.

Reply:
I can remove it - I wanted to address @crowlKats's concern.

Reply:
I agree with Luca... OTOH it will be hard to discover `location` API unless you plough through manual. I guess in this situation not showing warning is preferable

Reply:
I would suggest that we leave the warning for the time being. Extra warning is better than confusion. If people feel strongly we can remove it later.